### DISCUSSION TITLE
Blade: @repeat() and @endrepeat

### DISCUSSION THREAD

**Discussion by pilot-max** [laravel/framework#39570] [Ideas]
The current PHP style syntax for a loop is ugly and verbose. If we could use @repeat instead of @for and pass an integer value of the number of times to loop, it would make our code more readable and slightly quicker to write

`@repeat(10)`
`    The current value is {{ $loop->index }}`
`@endrepeat`

instead of

`@for ($i = 0; $i < 10; $i++)`
`    The current value is {{ $i }}`
`@endfor`

**Comment by shaedrich** [comment]
> ```blade
> @for ($i = 0; $i < 10; $i++)
>     The current value is {{ $i }}
> @endfor
> ```
Well, you can make it a little shorter:
```blade
@foreach(range(0, 9) as $i)
    The current value is {{ $i }}
@endforeach
```
